### PR TITLE
Qsw 794 - Initial changes for Dashboard DB

### PR DESCRIPTION
**Pull request by claudio525**
Has to go with https://github.com/ucgmsim/qcore/pull/74

### CODE REVIEW

File: dashboard/DashboardDB.py [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+            )
+        ).fetchall()
+
+        conn.close()
+        return [SQueueEntry(*result) for result in results]
+
+    def _create_queue_table(self, cursor, hpc: const.HPC):
+        # Add latest table
+        cursor.execute(
+            """CREATE TABLE {}(
```
Comment (jasonmotha) [inline dashboard/DashboardDB.py]: CREATE TABLE IF NOT EXISTS 

So it doesn't break if it already exists even though technically it never should.

File: dashboard/DashboardDB.py [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+        """Updates the daily core hours usage.
+
+        The core hour usage for a day is calculated as
+        last saved (for current day) TOTAL_CORE_USAGE - current total_core_usage.
+        """
+        # Do nothing if total_core_usage is None
+        if total_core_usage is None:
+            return
+
+        conn = sql.connect(self.db_file)
```
Comment (jasonmotha) [inline dashboard/DashboardDB.py]: Can we make a setup and teardown with and a 'with'? Seems like that would be the best practice.

Comment (sungeunbae) [reply dashboard/DashboardDB.py]: @jasonmotha  Can you show an example?

Comment (claudio525) [reply dashboard/DashboardDB.py]: have added this in commit   https://github.com/ucgmsim/slurm_gm_workflow/pull/68/commits/6c6224e377c429bcf175c4f0adeb242552af6a75

**Pull request merged**